dockstore-cli-integration-testing/generated/src/main/resources/pom.xml

```diff
@@ -19,7 +19,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.dockstore</groupId>
   <artifactId>dockstore-cli-integration-testing</artifactId>
-  <version>1.17.0</version>
+  <version>1.17.0-SNAPSHOT</version>
```

Comment:
(I may have asked this before, a while back)
The `SNAPSHOT` suffix generates the "non-release"-type artifact, and since we just released 1.17 and are now working on 1.18, would it be appropriate to bump the release number to 1.18 (`1.18.0-SNAPSHOT`)?